src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/GradleVerifier.java

```diff
@@ -67,11 +67,11 @@ public class GradleVerifier extends CodeVisitorSupport implements BuildSystemVer
     @Override
     public void verify(HostingRequest issue, HashSet<VerificationMessage> hostingIssues) throws IOException {
         GitHub github = GitHub.connect();
-        forkTo = issue.getRepositoryUrl();
-        forkFrom = issue.getNewRepoName();
+        forkFrom = issue.getRepositoryUrl();
```

Comment:
Haha oops

It was interesting to read as we return that exact same error in 3 different but similar flows